### PR TITLE
Add Currency, InputCurrency and FieldCurrency components

### PR DESCRIPTION
**Pull request by ivan-dalmet**
- [TextCurrency preview](https://start-ui-git-format-number-bearstudio.vercel.app/storybook/index.html?path=/docs/components-textcurrency--default)
- [InputCurrency preview](https://start-ui-git-format-number-bearstudio.vercel.app/storybook/index.html?path=/docs/components-inputcurrency--default)
- [FieldCurrency preview](https://start-ui-git-format-number-bearstudio.vercel.app/storybook/index.html?path=/docs/fields-fieldcurrency--default)

### CODE REVIEW

File: src/components/TextCurrency/docs.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { Stack, Code } from '@chakra-ui/layout';
```
Comment (yoannfleurydev) [inline src/components/TextCurrency/docs.stories.tsx:3]:
```suggestion
import { Stack, Code } from '@chakra-ui/react';
```

File: src/components/InputCurrency/docs.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+
+import { Stack, Code } from '@chakra-ui/layout';
```
Comment (yoannfleurydev) [inline src/components/InputCurrency/docs.stories.tsx:3]:
```suggestion
import { Stack, Code } from '@chakra-ui/react';
```

**Review by yoannfleurydev — CHANGES_REQUESTED** [review]

File: src/components/InputCurrency/docs.stories.tsx (hunk 2 above)
Comment (ivan-dalmet) [reply src/components/InputCurrency/docs.stories.tsx]: Fixed :)

**Review by yoannfleurydev — APPROVED** [review]
I approve this PR. I didn't see anything wrong, 

I didn't checkout on my machine, so if you want we can check together on monday, else you can merge this if you are confident enough.

**Pull request merged**